beacon-chain/rpc/eth/validator/validator.go

```diff
@@ -175,45 +187,59 @@ func (vs *Server) GetSyncCommitteeDuties(ctx context.Context, req *ethpbv2.SyncC
 		return nil, status.Error(codes.Unavailable, "Syncing to latest head, not ready to respond")
 	}
 
-	slot, err := slots.EpochStart(req.Epoch)
+	currentEpoch := slots.ToEpoch(vs.TimeFetcher.CurrentSlot())
+	lastValidEpoch := syncCommitteeDutiesLastValidEpoch(currentEpoch)
+	if req.Epoch > lastValidEpoch {
+		return nil, status.Errorf(codes.InvalidArgument, "Epoch is too far in the future. Maximum valid epoch is %v.", lastValidEpoch)
+	}
+
+	var stateEpoch types.Epoch
+	currentSyncCommitteeFirstEpoch, err := slots.SyncCommitteePeriodStartEpoch(currentEpoch / params.BeaconConfig().EpochsPerSyncCommitteePeriod)
+	if err != nil {
+		return nil, status.Errorf(codes.InvalidArgument, "Could not get sync committee period start epoch: %v.", err)
+	}
+	if req.Epoch >= currentSyncCommitteeFirstEpoch {
+		stateEpoch = currentSyncCommitteeFirstEpoch
+	} else {
+		stateEpoch, err = slots.SyncCommitteePeriodStartEpoch(req.Epoch)
+		if err != nil {
+			return nil, status.Errorf(codes.InvalidArgument, "Could not get sync committee period start epoch: %v.", err)
+		}
+	}
+	slot, err := slots.EpochStart(stateEpoch)
```

Comment:
Final feedback. No need for the if else condition.  Since we already verified `req.Epoch > lastValidEpoch`, the following should be work

```suggestion

	slot, err := slots.EpochStart(req.Epoch)
```